dist/npm.js

```diff
@@ -11065,7 +11065,7 @@ var require_readFile = __commonJS({
       value: true
     });
     exports.readFile = readFile;
-    exports.readFileSync = readFileSync4;
+    exports.readFileSync = readFileSync3;
```

Comment:
I'm guessing I don't have to constantly review the `.js` files since they are generated, right ?

Reply:
That's right, no need to review JS files in the `dist` folder 😛 It would be nice if there was an easy way to tell GitHub to exclude a folder from PRs.